assemble/bin/start-daemon.sh

```diff
@@ -103,9 +90,61 @@ ERRFILE="${ACCUMULO_LOG_DIR}/${SERVICE}_${LOGHOST}.err"
 rotate_log "$OUTFILE" ${ACCUMULO_NUM_OUT_FILES}
 rotate_log "$ERRFILE" ${ACCUMULO_NUM_OUT_FILES}
 
-# Fork the process, store the pid
-nohup ${NUMA_CMD} "$COMMAND" "${SERVICE}" --address "${ADDRESS}" >"$OUTFILE" 2>"$ERRFILE" < /dev/null &
-echo $! > ${PID_FILE}
+if [[ "$SERVICE" != "tserver" || $NUM_TSERVERS -eq 1 ]]; then
+   # Check the pid file to figure out if its already running.
+   PID_FILE="${ACCUMULO_PID_DIR}/accumulo-${ACCUMULO_IDENT_STRING}-${SERVICE}.pid"
+   if [ -f ${PID_FILE} ]; then
+      PID=`cat ${PID_FILE}`
+      if kill -0 $PID 2>/dev/null; then
+         # Starting an already-started service shouldn't be an error per LSB
+         echo "$HOST : $SERVICE already running (${PID})"
+         exit 0
+      fi
+   fi
+   echo "Starting $SERVICE on $HOST"
+
+   # Fork the process, store the pid
+   nohup ${NUMA_CMD} "$COMMAND" "${SERVICE}" --address "${ADDRESS}" >"$OUTFILE" 2>"$ERRFILE" < /dev/null &
```

Comment:
So, before this change (and 1 tserver per host) the user can influence NUMA options using (defined in config.sh):
# NUMA related environment variables
# ACCUMULO_ENABLE_NUMACTL   (Default: true) Use numactl if present on the system
# ACCUMULO_NUMACTL_OPTIONS  (Default: --interleave=all) options to the numactl command

The "if" uses these options. The "else" recalculates the options for each tserver using the TSERVER_NUMA_OPTIONS array. Seems like we could collapse these to one variable, but then you have to know when to make ACCUMULO_NUMACTL_OPTIONS an array and when not to.